internal/bminventory/inventory.go

```diff
@@ -2185,6 +2186,7 @@ func (b *bareMetalInventory) GetClusterInternal(ctx context.Context, params inst
 		// Clear this field as it is not needed to be sent via API
 		host.FreeAddresses = ""
 	}
+	cluster.HostCount = int32(len(cluster.Hosts))
```

Comment:
Aren't there other places in the code where cluster object is constructed? Shouldn't that property be properly filled when there are hosts associated with the cluster? For example: https://github.com/openshift/assisted-service/blob/73440529c68424ea318362ded9874042a9d2de8e/internal/bminventory/inventory.go#L4070

Developers in the future might expect it to contain correct value while writing new code.

Reply:
yes, thanks for pointing it out :smile: